pgzero/game.py

```diff
@@ -213,6 +212,55 @@ def get_draw_func(self):
                 )
             return draw
 
+    def _call_users_on_enter_func(self):
+        # Calls the user's on_enter function if defined.
```

Comment:
I feel like `on_enter()` means that "there is more than one way to do it" because we also promote using the module scope to set up the initial game state.

Reply:
I think an `on_enter()` function is useful for encapsulating start up code, but I understand your point.

Would you like me to remove it?